spatial/r3/mat_unsafe.go

```diff
@@ -0,0 +1,21 @@
+// Copyright ©2021 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build !safe
+// +build !safe
+
+package r3
+
+import "unsafe"
+
+const tagSafe = false
+
+func (m *Mat) setBackingSlice(vals []float64) {
+	copy(m.backingSlice(), vals)
```

Comment:
I did not perform this change as requested since it is only valid from go 1.17+ and the go.mod specifies 1.14. I've instead taken the pointer from `&vals[0]`

Reply:
The go.mod needs updating, but that won't fix this issue yet. Please add a todo `// TODO(kortschak): Use array conversion with go1.16 is no longer supported.`